src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4822,6 +4822,184 @@ .ForwardHelpCategory Cmdlet
 ";
         }
 
+#if UNIX
+        internal static string GetSudoFunctionText()
+        {
+            return @"
+Set-StrictMode -Version Latest
+
+$script:AmbiguousOperators = @('>', '<', '|', '&')
```

Comment:
Because built-in functions are not in a script file, `script:` implies global scope - which we don't want here.

Also - you don't need `@()` when the value is already an array (you use the `,` operator).